SD-K9/src/app/providers/pathfinder.service.ts

```diff
@@ -6,6 +6,8 @@ import { SVGManager } from './svg-manager.service';
 
 @Injectable()
 export class Pathfinder {
+  DISTANCE_BETWEEN_NODES = 8;
```

Comment:
Is it possible to use a formula? For instance,`area += yi+y(i+1) * (x(i+1)-xi)/2 ` by following the nest point in the svg path, and divide it by the number of nodes present in the svg file? The area can be the rectangle used to serve as background. Or will this affect performance?

Reply:
I agree, we should find a way to fix this. The problem with the area is that the nodes in the svg are at a specific distance from each other (5 pixels) and using the area would not work in this case since the value would not be accurate and could lead to neighbors being missed.

Reply:
I understand. In this case, is it possible to get the distance between two nodes then multiplying by 1.5? So it can recognize its immidiate neighboring node, if any?

Reply:
We can get the value from the svg floor plan meta-data, this way it can be customizable for each floor plan.